dbs/dbs.go

```diff
@@ -0,0 +1,29 @@
+package dbs
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+)
+
+var (
+	Db *sqlx.DB
+)
+
+func InitDb() {
+	_db, err := sqlx.Connect(
+		"mysql",
+		fmt.Sprintf("%s:%s@tcp(%s:%s)/%s?charset=utf8&parseTime=True&loc=Local",
+			os.Getenv("DB_USERNAME"),
+			os.Getenv("DB_PASSWORD"),
+			os.Getenv("DB_HOSTNAME"),
+			os.Getenv("DB_PORT"),
+			os.Getenv("DB_DATABASE")))
+	if err != nil {
+		log.Fatalf("Cannot Connect to Database: %s", err)
+	}
+	Db = _db
+}
```

Comment:
```suggestion
func GetDB() (*sqlx.DB, error) {
	return sqlx.Connect(
		"mysql",
		fmt.Sprintf("%s:%s@tcp(%s:%s)/%s?charset=utf8&parseTime=True&loc=Local",
			os.Getenv("DB_USERNAME"),
			os.Getenv("DB_PASSWORD"),
			os.Getenv("DB_HOSTNAME"),
			os.Getenv("DB_PORT"),
			os.Getenv("DB_DATABASE")))
}
```
他のパッケージがここに依存しているのはあまり嬉しくないのでDBを返す関数にしてしまって、それを`main.go`で依存として他のところに渡してあげるのが良さそうですね。